### ISSUE TITLE
Screen breaking up

### ISSUE THREAD

**Issue by paul1149** [MaartenBaert/ssr#619]
Hi,
Just started up with SSR, and it's very nice. I've used it twice, and got the output I wanted.

But each time the display has broken up seriously. I get various hidden windows appearing, and  the chosen rectangle being recording stealing focus, so that I am unable to reach the SSR window to control the session. I was able to stop the recording using Ctrl-r.

I'm on a Vista-era unit, with an Intel C2D, and 6gb Ram. I have an nvidia chip and use the nouveau driver.

Thanks.

**Comment by MaartenBaert** [comment]
Can you upload the recorded video? Does what shows up in the video match what you see on your screen, or is only the recording itself breaking up?

**Comment by paul1149** [comment]
Hi,
No, the video itself is pretty much what I expect. It's the whole computer display that is thrown into disarray. All kinds of windows appearing, inability to switch focus among programs. I'm on MX17, based on Debian 9.3 x64. Perhaps it's the nvidia chip.

**Comment by MaartenBaert** [comment]
That's really weird. Have you ever seen something like this with any other program? Can you try to [record your screen with the FFmpeg command-line tool](https://trac.ffmpeg.org/wiki/Capture/Desktop) to see what happens?

**Comment by paul1149** [comment]
I have been having some problems. It's an older system, with an nvidia card on linux, nouveau driver. Google earth seems to demand too much, and only a small working rectangle manifests in its window, and the window doesn't fully paint, showing the programs below it. I suspect the card is causing all this, am considering an upgrade. Thanks.

**Comment by MaartenBaert** [comment]
Give the proprietary NVIDIA driver a try first. Nouveau is just not very good.

**Comment by paul1149** [comment]
That was my first thought as well, but attempting to install it got me into a lot of trouble, and I ended up restoring the system from an image. That's why, rather than repeat that, I've been thinking of a hardware upgrade, to escape nvidia altogether.

**Comment by MaartenBaert** [comment]
My experience with GPUs and drivers on Linux is that the NVIDIA binary driver is the best option, however their open-source driver (Nouveau) is the worst option. For AMD, both the binary driver and the open-source driver are usable but they have several significant (but very different) issues, I don't recommend them at all. Intel uses the open-source driver as the official driver so there is only one option, but it is quite good. However the performance isn't great.

I'm pretty sure that this is just a problem with the driver, not the hardware. How did you install the binary driver?

**Comment by paul1149** [comment]
If I remember correctly, I used the MX package handler to install the nvidia driver. It gave me a choice of, I believe, three versions, and I took the latest. Other than the problem being discussed, I haven't had much nouveau problem, just a small amount of screen tear when scrolling. And to my eye I never saw a quality difference between the two drivers. But I will consider taking an up to the moment image, then trying the proprietary driver again.

**Comment by paul1149** [comment]
Hi Maarten,
I finally did the test. This time I took the vanilla version of the nvidia driver, using the MX driver utility. I think last time I took the latest one in debian backports. Installation went well. It said there was a conflict with the generic driver, and to reboot to solve it.

SSR said to disable flipping, which I did temporarily. Next time I'll make it permanent. There was absolutely no video breakup at all. Everything seems to be working fine. Also, the screen tearing I was getting at scrolling seems to be gone.

Thanks for your attention to my problem. Much appreciated.

BW
